packages/wallet/src/test-utils/launchNode.test.ts

```diff
@@ -0,0 +1,19 @@
+import { Provider } from '@fuel-ts/providers';
+
+import { WalletUnlocked } from '../wallets';
+
+import { launchNodeAndGetWallets } from './launchNode';
+
+/**
+ * This test is skipped because it requires fuel-core to be installed.
+ * TODO: Figure out a way to run this test using pnpm fuels-core in CI.
+ */
+test.skip('launchNodeAndGetWallets', async () => {
+  const { stop, provider, wallets } = await launchNodeAndGetWallets();
```

Comment:
We could add a new property to `LaunchNodeOptions`:

```ts
useSystemFuelCore: boolean; // default = false
```

Then use each binary accordingly:

```ts
    const command = (useSystemForc ? 'fuel-core' : './node_modules/.bin/fuels-core')
    const child = spawn(command, [
      'run',
      ...
```

We cannot use `pnpm` as the user may be using another package manager, so we refer to the binary directly from inside `node_modules/.bin` — I believe the `wallet` package must depend on `@fuel-ts/fuel-core` for this.

Reply:
FWIW, after syncing some requirements with @luizstacio, I'm also following this approach in Tooling PR.

Reply:
No I agree this is a great solution! We should both go ahead with this implementation. I am almost there. I'm facing some trouble because the cleanup logic that I have in place is not being able to cleanup the process started with the `pnpm` binary properly.

Reply:
I'm using something like this in the Tooling PR:

```ts
import kill from 'tree-kill';

const myProcess = spawn(...);

kill(Number(myProcess.pid));
```

Reply:
Thanks @arboleya, this worked!